packages/autocomplete-plugin-recent-searches/package.json

```diff
@@ -15,9 +15,9 @@
   },
   "scripts": {
     "build:clean": "rm -rf ./dist",
-    "build:esm": "babel src --root-mode upward --extensions '.ts,.tsx' --out-dir dist/esm  --ignore '**/*/__tests__/'",
+    "build:esm": "babel src --root-mode upward --extensions '.ts,.tsx' --out-dir dist/esm --ignore '**/*/__tests__/'  && cp src/style.css ./dist/esm/",
```

Comment:
I don't know what to do with the style.css file here, so I just copied it to `dist` on build at the moment.

Reply:
We'll think about this as a next step. It'd be nice to be able to do:

```js
import '@algolia/autocomplete-plugin-recent-searches/dist/style.css';

// Like DocSearch, with `root/style.js` importing `root/dist/style.css`
import '@algolia/autocomplete-plugin-recent-searches/style';
```

Reply:
Can you show me where it's done? I will apply it in a separate PR.
So far I've found only this: https://github.com/algolia/docsearch/blob/next/packages/docsearch-react/style/variables.js#L1:L1

Reply:
It happens there:

1. https://github.com/algolia/docsearch/blob/56ab94a6b8f445d69f0d63e93de7479a07f8e192/packages/docsearch-react/package.json#L15
2. https://github.com/algolia/docsearch/blob/56ab94a6b8f445d69f0d63e93de7479a07f8e192/packages/docsearch-react/style/index.js

Reply:
Hmm. I think I've done it: https://github.com/algolia/autocomplete.js/pull/326/commits/c00d92e4f3d34e9adc840c440798719007f9776e
Did I do it right? What is enabling this? `@babel/preset-env`?

Reply:
It looks good. That's the style loader of the bundler used that understands it (webpack, Parcel etc.).

Reply:
Ah okay. It's the consuming side which understands it.